samcli/lib/iac/utils/iac_project_resolver.py

```diff
@@ -0,0 +1,114 @@
+"""
+Provide IacProjectResolver to process IaC project information from command line options
+"""
+
+import logging
+import os
+
+from typing import List, Tuple, Optional
+from click import Context
+import click
+
+from samcli.lib.iac.interface import ProjectTypes, IacPlugin, Project
+from samcli.lib.iac.utils.helpers import get_iac_plugin
```

Comment:
is here the only place we use `get_iac_plugin`? Maybe we can move it in this file and refactor it as an instance method of `IacProjectResolver`, and get rid of the `helpers.py` file.

Reply:
Done.